### ISSUE TITLE
"Alexander von Humboldt Briefedition" is (now) part of ../251

### ISSUE THREAD

**Issue by cthomasdta** [gfranzini/digEds_cat#334]
Dear Greta, dear all, the "Alexander von Humboldt Briefedition", https://dig-ed-cat.acdh.oeaw.ac.at/editions/detail/220 is now part of "Edition Humboldt Digital", https://dig-ed-cat.acdh.oeaw.ac.at/editions/detail/251. Therefore, I would not bother to add info or correct minor inaccuracies in ../220. Instead, I will update the entry for "Edition Humboldt Digital", ../251.

Since you have already handle PIDs in place, how do you deal with entries like ../220 that should (imo) rather be removed than updated?

**Comment by gfranzini** [comment]
Hi Christian, sincere apologies for the huge delay but the catalogue's had to become a spare time activity for me, sadly.
What we would normally do for "expired" editions is add a red label stating that the project is no longer available at that address but your case is different. I would personally keep it and add a different label with a redirect to 221 to bypass PID complications, if any, but maybe @csae8092 has a better solution?

**Comment by cthomasdta** [comment]
I am fine with this, but I would think that it would be in your (at least Gretas) interest to keep the original entry, but mark it as 'outdated', 'deprecated' or the like. Your catalogue should IMO have a way of dealing with this not-too-seldom phenomenon of data moving from one edition (in)to another, being updated and/or converted into another format along the way of integration and from then on being cared for only in the new environment.
But the decision is entirely yours, of course, I am fine with the merger.

**Comment by gfranzini** [comment]
Yes, I would prefer to keep it and mark it as deprecated. We've kept and flagged other editions which are no longer available, and while @csae8092 is right in pointing out the statistics issue, our online documentation is quite clear with regard to outdated editions so I would expect users to take these into account.

So, in short: keep 220, mark it as deprecated and redirect its PID to 221. 

Doable, @csae8092 ?

**Comment by csae8092** [comment]
I redirected the handle and changed (only in the database) the availibility value; https://dig-ed-cat.acdh.oeaw.ac.at/editions/detail/220 (maybe we should change the text "Edition no longer available under provided link" to something like "Edition no longer available and/or the entry is no longer maintained"

**Comment by gfranzini** [comment]
> "Edition no longer available and/or the entry is no longer maintained"
 
That would be great, thanks! :)

**Comment by csae8092** [comment]
> BTW, "XML not used" is not true for ../220, although you can't see it, there has been TEI-XML underneath from the start. But like I said in #issue-586919040, I would not bother to add info or correct minor inaccuracies in ../220.

@cthomasdta feel free to correct infos about here, than I'll update the [corresponding entry](https://dig-ed-cat.acdh.oeaw.ac.at/entry-220.html) accordingly. Thanks for you patience!
